R/zchunk_L222.en_transformation_USA.R

```diff
@@ -67,210 +75,575 @@ module_gcam.usa_L222.en_transformation_USA_DISABLED <- function(command, ...) {
     L222.GlobalTechCost_en <- get_data(all_data, "L222.GlobalTechCost_en")
     L222.GlobalTechShrwt_en <- get_data(all_data, "L222.GlobalTechShrwt_en")
     L222.GlobalTechCapture_en <- get_data(all_data, "L222.GlobalTechCapture_en")
-    L222.GlobalTechShutdownProfit_en <- get_data(all_data, "L222.GlobalTechShutdownProfit_en")
-    L222.GlobalTechShutdown_en <- get_data(all_data, "L222.GlobalTechShutdown_en")
-    L222.GlobalTechSCurveProfit_en <- get_data(all_data, "L222.GlobalTechSCurveProfit_en")
+    # L222.GlobalTechShutdownProfit_en <- get_data(all_data, "L222.GlobalTechShutdownProfit_en")
```

Comment:
Why are we commenting these out?

Reply:
I don't really understand the details but it is related to this
https://github.com/JGCRI/gcamdata/issues/797

Reply:
Essentially, they're either inputs that aren't created with the current regional markets settings, or they're created but just completely empty

Reply:
Got it. Thanks.